### ISSUE TITLE
Use local scripts rather than remote?

### ISSUE THREAD

**Issue by slackhead** [prologic/gopherclient#16]
Hi :)

Instead of loading a file remotely from jquery.com in template.go, would it be possible to keep a copy locally?

1) It would save some time for people on slow internet.
2) Some people might have jquery.com blocked in their hosts file because of 1), or other reasons.

Thanks :)

**Comment by winwisely99** [comment]
I am also wondering about that.

I want to embed the web files inside as part of the build

**Comment by prologic** [comment]
Yes it is possible. I typically use [go.rice](https://github.com/GeertJohan/go.rice) for such things on many of my projects. So if someone would like to put up a PR for this I'd happily review, accept and merge it :)
